src/util/numeric.h

```diff
@@ -288,7 +288,7 @@ bool isBlockInSight(v3s16 blockpos_b, v3f camera_pos, v3f camera_dir,
 */
 inline s32 myround(f32 f)
 {
-	return floor(f + 0.5);
+	return f < 0.f ? ceil(f - 0.5f) : floor(f + 0.5f);
```

Comment:
ceil and floor are not required (simply cast to s32 which drops the decimal part yielding the same result). Because the function is returning s32 and not double (as C round() does) it's probably safer to explicitly cast rather than relying on the implicit cast (as if by assignment) that occurs when returning the value. E.g. `return f < 0.f ? (s32)(f - 0.5f) : (s32)(f + 0.5f);`

I hope we use myround() everywhere instead of round() otherwise there will be different results on platforms that use C's round()...

Is there a unit test for this function? Not sure how this bug hung around for so long :)